data/locale/jack_mixer-fr.po

```diff
@@ -0,0 +1,677 @@
+
+#: jack_mixer/app.py:475 jack_mixer/app.py:557 jack_mixer/app.py:1138
+#: jack_mixer/app.py:478 jack_mixer/app.py:560 jack_mixer/app.py:1144
+#: jack_mixer/app.py:479 jack_mixer/app.py:561 jack_mixer/app.py:1145
+#, python-brace-format
+msgid "Error loading project file '{filename}': {msg}"
+msgstr "Erreur lors du chargement du fichier projet '{filename}': {msg}"
+
+#: jack_mixer/app.py:542 jack_mixer/app.py:545 jack_mixer/app.py:546
+msgid "XML files"
+msgstr "Fichiers XML"
+
+#: jack_mixer/app.py:546 jack_mixer/app.py:549 jack_mixer/app.py:550
+msgid "All files"
+msgstr "Tous les fichiers"
+
+#: jack_mixer/app.py:567 jack_mixer/app.py:570 jack_mixer/app.py:571
+msgid "Open project"
+msgstr "Ouvrir un projet"
+
+#: jack_mixer/app.py:615 jack_mixer/app.py:658 jack_mixer/app.py:618
+#: jack_mixer/app.py:661 jack_mixer/app.py:619 jack_mixer/app.py:662
+#, python-brace-format
+msgid "Error saving project file '{filename}': {msg}"
+msgstr "Erreur lors de la sauvegarde du fichier projet '{filename}': {msg}"
+
+#: jack_mixer/app.py:622 jack_mixer/app.py:625 jack_mixer/app.py:626
+msgid "Save project"
+msgstr "Sauvegarder le projet"
+
+#: jack_mixer/app.py:675 jack_mixer/app.py:678 jack_mixer/app.py:679
+msgid "<b>Quit application?</b>"
+msgstr "<b>Quitter l'application ?</b>"
+
+#: jack_mixer/app.py:678 jack_mixer/app.py:681 jack_mixer/app.py:682
+msgid ""
+"All jack_mixer ports will be closed and connections lost,\n"
+"stopping all sound going through jack_mixer.\n"
+"\n"
+"Are you sure?"
+msgstr ""
+"Tous les ports de jack_mixer seront fermés et les connexions perdues,\n"
+"ce qui arrêtera tout son passant par jack_mixer.\n"
+"\n"
+"Êtes-vous sûr ?"
+
+# Don't translate this unless you want default channel names to be localized
+#: jack_mixer/app.py:743 jack_mixer/app.py:746 jack_mixer/app.py:747
+msgid "Input"
+msgstr "Entrée"
+
+# Don't translate this unless you want default channel names to be localized
+#: jack_mixer/app.py:746 jack_mixer/app.py:749 jack_mixer/app.py:750
+msgid "Output"
+msgstr "Sortie"
+
+#: jack_mixer/app.py:874 jack_mixer/app.py:877 jack_mixer/app.py:878
+msgid "Are you sure you want to clear all channels?"
+msgstr "Êtes-vous sûr de vouloir nettoyer tous les canaux ?"
+
+#: jack_mixer/app.py:1103 jack_mixer/app.py:1109 jack_mixer/app.py:1110
+msgid "load mixer project configuration from FILE"
+msgstr "charger la configuration du projet de mixeur à partir du FICHIER"
+
+#: jack_mixer/app.py:1110 jack_mixer/app.py:1116 jack_mixer/app.py:1117
+msgid "enable debug logging messages"
+msgstr "activer les messages de journalisation de débogage"
+
+#: jack_mixer/app.py:1128 jack_mixer/app.py:1134 jack_mixer/app.py:1135
+msgid ""
+"Mixer creation failed:\n"
+"\n"
+"{}"
+msgstr ""
+"Impossible de créer le mixeur :\n"
+"\n"
+"{}"
+
+# Don't translate this unless you want JACK output port names to be localized
+#: jack_mixer/channel.py:90
+#, python-brace-format
+msgid "{channel_name} Out"
+msgstr "{channel_name} Sortie"
```

Comment:
See comment for line 46.

(Though, this string isn't actually used atm for the naming of the output port, but this is a bug.)

Reply:
Nevermind, this string is for the default *channel names* not the *port* names.

Reply:
Fixed the issue mentioned in parentheses above in 4b318c2a.

Reply:
_same than above_